### ISSUE TITLE
CRD YAML conflict when when going through Gateway Connectivity doc

### ISSUE THREAD

**Issue by archelangelo** [istio/istio#14792]
**Bug description**
In [Gateway Connectivity](https://istio.io/docs/setup/kubernetes/install/multicluster/gateways/) doc, the following command generates a conflict in the YAML file:
```
cat install/kubernetes/helm/istio-init/files/crd-* > $HOME/istio.yaml
```
As the `install/kubernetes/helm/istio-init/files/crd-certmanager-*.yaml` files don't have the `---` separator at the end, the `cat` command appends two YAMLs together without separation. This causes a missing CRD `certificates.certmanager.k8s.io` in the cluster. This missing CRD didn't cause any issue in my demo app, but may do that in the future.

**Affected product area (please put an X in all that apply)**

[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience

**Expected behavior**
YAML file having no conflict.

**Steps to reproduce the bug**
Run
```
cat install/kubernetes/helm/istio-init/files/crd-* > $HOME/istio.yaml
```

**Version (include the output of `istioctl version --remote` and `kubectl version`)**
Irrelevant.

**How was Istio installed?**
Irrelevant.

**Environment where bug was observed (cloud vendor, OS, etc)**
Irrelevant.

**Comment by sdake** [comment]
Thank you for the report.  I will do what I can to get this backported to 1.1, and this has just now been merged to 1.2/master.

Cheers
-steve

**Comment by sdake** [comment]
https://github.com/istio/istio/pull/14844

**Comment by linsun** [comment]
reduce to p1 since fix is already in 1.2

**Comment by sdake** [comment]
closing issue as problem is resolved.
